.dockerignore

```diff
@@ -0,0 +1,3 @@
+/.devcontainer
+/.github
+/.vscode
```

Comment:
this would remove vscode dev environment and pipelines :)

Reply:
Okay, will fix this